NAMING_CONVENTIONS.md

```diff
@@ -0,0 +1,37 @@
+Arnold USD naming conventions
+=======================
+
+The Arnold USD implementation supports arnold-specific attributes through custom schemas.
+
+## Arnold attributes on USD nodes
+
+Additional arnold attributes on USD-native nodes (geometries, lights, etc...) need to be in the "arnold" namespace
+
+Example:
+def DomeLight "my_light"
+{
+ float intensity=2
+ int arnold:max_bounces=99
+}
+
+## Arnold Shaders
+
+Arnold shaders are supported with UsdShade Shader primitives. The info:id attribute should be the shader name, with an "arnold" namespace. The attributes will be in the "inputs" scope.
```

Comment:
I don't think we need to specify that shader parameters have to be prefixed with `inputs:` that's part of the UsdShadeShader schema, and considered a standard.

Reply:
Maybe the "best of both world" is to keep it documented but mention it is the UsdShadeShader standard practice, potentially with a link to it that elaborates it. Or would that end up being too verbose? At this stage with it being so new for many I think even the smallest hints to reasoning on implementation/workflow could be helpful.

Reply:
Sounds good to me, a link to the UsdShadeShader could be useful.

Reply:
done. Let me know if more details could be useful.